iset.mm

```diff
@@ -148169,6 +148242,320 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
+    ( crp wcel c1 cap wbr wa clogb co clog cdiv wceq rplogbval 3anidm13 relogcl
+    cfv cc recnd adantr logrpap0 dividapd eqtrd ) ABCZADEFZGZAAHIZAJPZUGKIZDUCU
+    DUFUHLAAMNUEUGUCUGQCUDUCUGAORSATUAUB $.
+
+  $( The logarithm of ` 1 ` to an arbitrary base ` B ` is 0.  Property 1(b) of
+     [Cohen4] p. 361.  See ~ log1 .  (Contributed by Stefan O'Rear,
+     19-Sep-2014.)  (Revised by Thierry Arnoux, 27-Sep-2017.) $)
+  rplogb1 $p |- ( ( B e. RR+ /\ B =//= 1 ) -> ( B logb 1 ) = 0 ) $=
+    ( crp wcel c1 cap wbr wa clogb co clog cfv cdiv cc0 wceq 1rp rplogbval log1
+    mp3an3 oveq1i simpl relogcld recnd logrpap0 div0apd syl5eq eqtrd ) ABCZADEF
+    ZGZADHIZDJKZAJKZLIZMUGUHDBCUJUMNOADPRUIUMMULLIMUKMULLQSUIULUIULUIAUGUHTUAUB
+    AUCUDUEUF $.
+
+  $( The general logarithm of a number to the base being Euler's constant is
+     the natural logarithm of the number.  Put another way, using ` _e ` as the
+     base in ` logb ` is the same as ` log ` .  Definition in [Cohen4] p. 352.
+     (Contributed by David A. Wheeler, 17-Oct-2017.)  (Revised by David A.
+     Wheeler and AV, 16-Jun-2020.) $)
+  rpelogb $p |- ( A e. RR+ -> ( _e logb A ) = ( log ` A ) ) $=
+    ( crp wcel ceu clogb co clog cfv cdiv c1 cap wbr wceq epr rplogbval mp3an12
+    eap1 loge oveq2i relogcl recnd div1d syl5eq eqtrd ) ABCZDAEFZAGHZDGHZIFZUGD
+    BCDJKLUEUFUIMNQDAOPUEUIUGJIFUGUHJUGIRSUEUGUEUGATUAUBUCUD $.
+
+  $( Change of base for logarithms.  Property in [Cohen4] p. 367.  (Contributed
+     by AV, 11-Jun-2020.) $)
+  rplogbchbase $p |- ( ( ( A e. RR+ /\ A =//= 1 )
+                  /\ ( B e. RR+ /\ B =//= 1 ) /\ X e. RR+ )
+                     -> ( A logb X ) = ( ( B logb X ) / ( B logb A ) ) ) $=
+    ( crp wcel c1 cap wbr wa clog cdiv clogb relogcld recnd logrpap0d rplogbval
+    cfv co wceq syl3anc simp1l simp2l simp1r simp2r divcanap7d oveq12d 3eqtr4rd
+    w3a simp3 ) ADEZAFGHZIZBDEZBFGHZIZCDEZUHZCJQZBJQZKRZAJQZUSKRZKRURVAKRZBCLRZ
+    BALRZKRACLRZUQURVAUSUQURUQCULUOUPUIZMNUQVAUQAUJUKUOUPUAZMNUQUSUQBULUMUNUPUB
+    ZMNUQAVHUJUKUOUPUCZOUQBVIULUMUNUPUDZOUEUQVDUTVEVBKUQUMUNUPVDUTSVIVKVGBCPTUQ
+    UMUNUJVEVBSVIVKVHBAPTUFUQUJUKUPVFVCSVHVJVGACPTUG $.
+
+  $( Value of the general logarithm with integer base.  (Contributed by Thierry
+     Arnoux, 27-Sep-2017.) $)
+  relogbval $p |- ( ( B e. ( ZZ>= ` 2 ) /\ X e. RR+ )
+        -> ( B logb X ) = ( ( log ` X ) / ( log ` B ) ) ) $=
+    ( c2 cuz cfv wcel crp wa c1 cap wbr clogb co clog cdiv wceq cc0 wne adantr
+    cz zgt1rpn0n1 simp1d simp3d wb eluzelz zapne syl2anc mpbird simpr rplogbval
+    1zzd syl3anc ) ACDEFZBGFZHAGFZAIJKZUNABLMBNEANEOMPUMUOUNUMUOAQRZAIRZAUAZUBS
+    UMUPUNUMUPURUMUOUQURUSUCUMATFITFUPURUDCAUEUMUKAIUFUGUHSUMUNUIABUJUL $.
+
+  $( Closure of the general logarithm with a positive real base on positive
+     reals.  (Contributed by Stefan O'Rear, 19-Sep-2014.)  (Revised by Thierry
+     Arnoux, 27-Sep-2017.) $)
+  relogbclap $p |- ( ( B e. RR+ /\ X e. RR+ /\ B =//= 1 )
```

Comment:
This is the same as ~ rplogbcl, isn't it?

Reply:
So it is. Will fix.